app/renderer/ui/components/main/sections/wallet/tabs/receive/index.tsx

```diff
@@ -37,7 +48,12 @@ class TabReceive extends TabComponent<any> {
                 <InputDefault />
                 <label className={styles.label}> Expires </label>
                 <InputDefault />
-                <ButtonDefault className={styles.submit}>SAVE AND GENERATE ADDRESS</ButtonDefault>
+                <ButtonDefault
+                  className={styles.submit}
+                  onClick={this.props.services.showUnimplementedMessage}
```

Comment:
An error happens to me when testing Sheikah and clicking on the "Receive tab" (checkout master and rebase against your branch). It says: `Uncaught TypeError: Cannot read property 'showUnimplementedMessage' of undefined
    at ProxyComponent.module.exports../app/renderer/ui/components/main/sections/wallet/tabs/receive/index.tsx.TabReceive.render (index.tsx:53)`